index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script  src="js/index.js"></script>
+    <meta charset="UTF-8">
+    <title>Nelson and Ahmed</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar" class="nav">
+      <ul class="nav-list">
+        <li>
+          <a href="#welcome-section">About</a>
+        </li>
+        <a id="profile-link" href="https://github.com/borrnelsonk" target="_blank" > Git Profile </a>  
+        </li>  
+        <li>
+          <a href="#projects">Work</a>
+        </li>
+        <li>
+          <a href="#contact">Contact</a>
+        </li>
+      </ul>
+    </nav>
+    
+
+ <section id="welcome-section" class="welcome-section">
+      <h1>Hello welcome to Ahmed and Nelsons page</h1>
+      <p>We are new web developers</p>
+    </section>
+     <section id="projects" class="projects-section">
+      <h2 class="projects-section-header">These are some of our projects</h2>
+
+      <div class="projects-grid">
+        <a
+          href="#"
+          target="_blank"
+          class="project project-tile"
+        >
+          
+          <p class="project-title">
+            <span class="code">&lt;</span>
+            football web page
+            <span class="code">&#47;&gt;</span>
+          </p>
+        </a>
+      </div>
+      
+       <a
+          href="#"
+          target="_blank"
+          class="project project-tile"
+        >
+          <img
+            class="project-image"
+            src="images/football.jfif"
+            alt="project"
+          />
+          <p class="project-title">
+            <span class="code">&lt;</span>
+            fish project
+            <span class="code">&#47;&gt;</span>
+          </p>
+        </a>
+        <a
+          href="#"
+          target="_blank"
+          class="project project-tile"
+      >
+         <img
+            class="project-image"
+            src="images/fish.jfif"
+            alt="project"
+            
```

Comment:
Unclosed image tag. Kindly check your file and close all opened elements or image tags.